File: 0001 Code Chef/0009 Circket World Cup Qualifier/implementation/0004 java/CircketWorldCup.java
import java.util.Scanner;

public class CircketWorldCup{
    public static void main(String[] args){
        Scanner console = new Scanner(System.in);
        String stringInput = console.nextLine();
        int score = Integer.parseInt(stringInput);
        if(score >= 12){
            System.out.println("YES");
        }else{
            System.out.println("NO");
        }
    }
}

File: 0001 Code Chef/0009 Circket World Cup Qualifier/implementation/0003 c#/CircketWorldCup.cs
using System;

class CircketWorldCup{
    static void Main(string[] args){
        string stringInput = Console.ReadLine();
        int score = Convert.ToInt32(stringInput);
        if(score >= 12){
            Console.WriteLine("YES");
        }else{
            Console.WriteLine("NO");
        }
    }
}